### PR TITLE
TUP-25202 can't get ambari configuration when I select retrieve it fromjob server

### PR DESCRIPTION
**Pull request by kjwang-talend**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TUP-25202

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/140/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/140/)

File: components/components-hadoopcluster/hadoopcluster-runtime/src/main/java/org/apache/ambari/api/TextJacksonJsonProvider.java [hunk 1]
```diff
@@ -68,4 +68,23 @@ protected boolean isJsonType(MediaType mediaType) {
...
         return true;
     }
+    
+    protected boolean hasMatchingMediaType(MediaType mediaType)
+    {
+    	if (super.hasMatchingMediaType(mediaType)) {
+    		return true;
+    	}
+    	
+        if (mediaType != null) {
```
Comment (wwang-talend) [inline components/components-hadoopcluster/hadoopcluster-runtime/src/main/java/org/apache/ambari/api/TextJacksonJsonProvider.java]: format is wrong

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/146/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/146/)

**Comment by kjwang-talend** [comment]
Close this one since it not usefull now.

**Pull request closed without merging**